mlflow/fastai.py

```diff
@@ -74,16 +74,12 @@ def get_default_conda_env(include_cloudpickle=False):
 
     import fastai
 
-    pip_deps = None
+    pip_deps = ["fastai={}".format(fastai.__version__)]
     if include_cloudpickle:
         import cloudpickle
 
-        pip_deps = ["cloudpickle=={}".format(cloudpickle.__version__)]
-    return _mlflow_conda_env(
-        additional_conda_deps=["fastai={}".format(fastai.__version__)],
-        additional_pip_deps=pip_deps,
-        additional_conda_channels=None,
-    )
+        pip_deps.append("cloudpickle=={}".format(cloudpickle.__version__))
+    return _mlflow_conda_env(additional_pip_deps=pip_deps, additional_conda_channels=None,)
```

Comment:
```suggestion
    return _mlflow_conda_env(additional_pip_deps=pip_deps)
```

@WeichenXu123 Could you clean up useless arguments like this one?